canvas_project/templates/settings.html

```diff
@@ -372,4 +383,44 @@ <h5 class="modal-title" id="messageModalLabel">Messages</h5>
                 }
             });
     });
+
```

Comment:
```suggestion
</script>
```
Because right now it doesn't work because the tag isn't closed